settings.gradle.kts

```
@@ -19,6 +19,8 @@ buildscript {
                 or(
                     // improved reporting
                     "IterableContainsInOrderOnly.*Spec",
+                    "IterableContainsInAnyOrder.*error cases.*",
+                    "IterableContainsInAnyOrderOnlyEntriesAssertionsSpec.*nullable cases.*",
```

Sorry, I informed you wrong, there is no need for the .* at the end as there is one already in line 32
```suggestion
                    "IterableContainsInAnyOrder.*error cases",
                    "IterableContainsInAnyOrderOnlyEntriesAssertionsSpec.*nullable cases",
```